recipes/rpdb/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "rpdb" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5d1a1cee34378ab075879dc30fa6328d448a9f680a66c4e84cac7382ad92f15f
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - rpdb
+
+about:
+  home: http://tamentis.com/projects/rpdb
+  license: ISC (ISCL)
+  license_family: OTHER
+  license_file: 
```

Comment:
Please include the license file, see https://conda-forge.org/docs/meta.html#packaging-the-license-manually if needed

Reply:
Done (used license file from https://github.com/tamentis/rpdb/blob/master/LICENSE.txt)